pages/common/git-stash.md

```diff
@@ -4,24 +4,24 @@
 
 - Stash current changes (except new files):
 
-`git stash save {{optional_stash_name}}`
+`git stash save {{optional_stash_message}}`
 
 - Include new files in the stash (leaves the index completely clean):
 
-`git stash save -u {{optional_stash_name}}`
+`git stash save -u {{optional_stash_message}}`
 
 - List all stashes:
```

Comment:
A second though about the "with" in the sentence above, couldn't it give the wrong impression of "List all stashes _that contain_ name, related branch and message"?

Maybe "List all stashes (shows stash name, related branch and message)"?